### PR TITLE
Addinformation

### PR DESCRIPTION
**Pull request by therealchrismcfly**
I added Styled Components for the exercise image and exercise description. Both components are now rendered at the index.js and the planner.js page!

### CODE REVIEW

File: src/components/CardLists/index.js [hunk 1]
```diff
@@ -16,17 +19,27 @@ function CardlistIndex() {
...
+											exercise.image,
+											exercise.description
+										);
 									}}
 								>
 									{'add to planner'}
 								</StyledCalendarButton>
 							</StyledCardHeader>
+							<StyledCardBody>
+								<StyledCardImage src={exercise.image}></StyledCardImage>
```
Comment (JessicaLoers) [inline src/components/CardLists/index.js:38]: To use a` <image />` element is not wrong but I would suggest you to use the `<Image />  component of next js`, which brings some advantages. 
Here's a link to the documentation: [next/image](https://nextjs.org/docs/api-reference/next/image) 👀

File: src/components/CardImage/styled.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import styled from 'styled-components';
+
+const StyledCardImage = styled.img`
```
Comment (JessicaLoers) [inline src/components/CardImage/styled.js:3]: please have a look at my comment regarding the `<Image />` Component of next js. ✌️

File: src/components/CardLists/planner.js [hunk 3]
```diff
@@ -27,6 +30,10 @@ function CardlistPlanner() {
 									delete
 								</StyledDeleteButton>
 							</StyledCardHeader>
+							<StyledCardBody>
+								<StyledCardImage src={card.image}></StyledCardImage>
```
Comment (JessicaLoers) [inline src/components/CardLists/planner.js:34]: To use a` <image />` element is not wrong but I would suggest you to use the `<Image />  component of next js`, which brings some advantages. 
Here's a link to the documentation: [next/image](https://nextjs.org/docs/api-reference/next/image)

**Review by JessicaLoers — COMMENTED** [review]
**Hi @therealchrismcfly**  ✌️
good job. I have just a minor note about the image. 

In general I'm quite sure that there're css rules you could get rid of if (e.g. some margin properties). 
But I now, is a work in progress and maybe a task for the future, when you're planning to finalize your Design. 

⭐️ **Working process hint of the day** 
Use the devtools to check which properties are redundant or really necessary.

File: src/components/CardLists/planner.js (hunk 3 above)
Comment (therealchrismcfly) [reply src/components/CardLists/planner.js]: thanks a lot for your feedback, I can learn so much from it! 🥳

**Comment by j-eick** [comment]
Well done. 👍🏼  
If I was a Korinthenkacker, then I'd say keep an eye on some indentations for better readability, but aside from that... good job!

File: src/components/CardLists/index.js [hunk 4]
```diff
@@ -16,17 +20,32 @@ function CardlistIndex() {
 			<StyledCardlist>
 				{exercises.map(exercise => {
 					return (
-						<StyledCard key={exercise._id}>
+						<StyledCard key={exercise.id}>
 							<StyledCardHeader>
-								<StyledHeadline>{exercise.name}</StyledHeadline>
+								<StyledCardHeadline>{exercise.name}</StyledCardHeadline>
```
Comment (j-eick) [inline src/components/CardLists/index.js:25]: Identations like this

**Review by j-eick — APPROVED** [review]
Good job! 👌🏼 

(I'm figuring out where to write what during the review, so not sure where this mesage is gonna pop up.)

**Pull request merged**